docker_entrypoint.sh

```diff
@@ -32,6 +32,8 @@ openssl x509 -outform der -in /mnt/cert/control.cert.pem -out /root/.lnd/start9/
 cat /root/.lnd/start9/control.cert.der | basenc --base64url -w0 > /root/.lnd/start9/control.cert.pem.base64url
 cp /mnt/cert/control.cert.pem /root/.lnd/tls.cert
 cp /mnt/cert/control.key.pem /root/.lnd/tls.key
+cp /mnt/cert/control.cert.pem /root/.lnd/public/tls.cert
+cp /mnt/cert/control.key.pem /root/.lnd/public/tls.key
 sed -i 's/\(BEGIN\|END\) PRIVATE KEY/\1 EC PRIVATE KEY/g' /root/.lnd/tls.key
```

Comment:
Does this file's path need to change to `/root/.lnd/public/tls.key`
If not, (and apparently not due to our testing so far) maybe tls.key really shouldn't go in the public dir.
tls.key I presume is for signing and decrypting connections to LND's rpc and/or p2p port.  Other services should probably not have access to it.  But it would make sense for them to have access to the .cert -- since likely in most scenarios the key is self-signed and that's a good way for them to know they're connecting to the right endpoint if the cert matches.